oblt-cli/setup/README.md

```diff
@@ -0,0 +1,31 @@
+# oblt-cli/setup
+
+Setup oblt-cli for use in GitHub Actions workflows. Only works in conjunction with the [google/auth](../../google/auth) action
+because it's required to authenticate with the Google Cloud Platform to access the OBLT cluster secrets
+
+## Inputs
+
+| name            | Description                                                          | required | default            |
+|-----------------|----------------------------------------------------------------------|----------|--------------------|
+| `github-token`  | The GitHub token with permissions fetch releases.                    | `true`   |                    |
+| `slack-channel` | The slack channel to be configured in the oblt-cli.                  | `false`  | `#observablt-bots` |
+| `username`      | Username to show in the deployments with oblt-cli, format: [a-z0-9]. | `false`  | `obltmachine`      |
+| `version`       | Install a specific version of oblt-cli. Latest version if empty.     | `false`  | `""`               |
+
+
+## Usage
+
+
+```yaml
+jobs:
+  run-oblt-cli:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: elastic/oblt-actions/google-auth@v1
+      - uses: elastic/oblt-actions/git/setup@v1
```

Comment:
What's the reason for this?

Reply:
https://github.com/elastic/oblt-actions/commit/2fd25ba23396799914cf18196294165d9e498aac#r142442505

Reply:
IIRC, oblt-cli would not work correctly without this.

The same action is used in the current example: https://github.com/elastic/apm-pipeline-library/tree/main/.github/actions/setup-oblt-cli

but I will add integration tests similar to `test-oblti-cli-*yml` in apm-pipeline-library to validate this add update the docs accordingly.

Reply:
> By default, `oblt-cli` uses SSH to perform Git operations, which is the recommended way; however, HTTP is supported. To configure `oblt-cli` to use HTTP use the flag `--git-http-mode` in the configure command:


> `oblt-cli configure --slack-channel=@MYSLACKMEMBERID --username=myusername --git-http-mode`
To perform git operations using HTTP a valid [GitHub access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and [Caching your GitHub credentials in Git](https://docs.github.com/en/github-ae@latest/get-started/getting-started-with-git/caching-your-github-credentials-in-git) is needed or alternatively: [Updating credentials from the macOS Keychain](https://docs.github.com/en/github-ae@latest/get-started/getting-started-with-git/updating-credentials-from-the-macos-keychain)

I guess, we could implicitly add the `elastic/oblt-actions/git/setup@v1` in the `setup-oblt` action?

Reply:
I will validate first if it's actually needed or if it's just some leftover.

I will add the tests while adding the other oblt-cli related actions.